Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -142,6 +148,13 @@ QString DMainWindow::ShowFileDialog()
 		.arg(SL("*.gcm *.iso *.ciso *.gcz *.wbfs *.elf *.dol *.dff *.tmd *.wad")));
 }
 
+QString DMainWindow::ShowFolderDialog()
+{
+	return QFileDialog::getExistingDirectory(this, tr("Browse for a directory to add"),
+                                             QDir::homePath(),
+                                             QFileDialog::ShowDirsOnly);
+}
```

Comment:
aye, the issue is that the line needs to be indented to the same location as the "return QFil..." bit and then spaces the rest of the way.